File: 3-Repetitions/sol.java
import java.util.*;
import java.lang.*;
import java.io.*;

class sol{
    // Comparator<> compare = new Comparator<>() {
    //     @Override
    //     public int compare( a, b) {
    //         return;
    //     }
    // };
    public static void  solve(){
        Scanner sc = new Scanner(System.in);
        String dna = sc.next();

        long i = 1, sol = 1, curr = 1;
        while(i<dna.length()){
            if(dna.charAt((int)i-1) == dna.charAt((int)i)){
                curr++;
            }else{
                if(curr > sol){
                    sol = curr;
                }
                curr = 1;
            }
            i++;
        }
        if(curr > sol){
            sol = curr;
        }

        System.out.println(sol);
    }
    public static void main (String[] args) throws java.lang.Exception
	{
        Scanner sc=new Scanner(System.in);
		int t=1;
        // t = sc.nextInt();
        while(t>0){
            solve();
            t--;
        }
    }
}

File: 3-Repetitions/sol.cpp
#include <bits/stdc++.h>
using namespace std;

#define deb(x) cout << #x << "=" << x << endl
#define ll long long
void solve()
{
	string dna;
	cin >> dna;
	ll sol = 1,i=1,curr = 1;
	while(i < dna.length()){
		if( dna[i-1] == dna[i]){
			curr++;
		}else{
			sol = max(curr,sol);
			curr = 1;
		}
		i++;
	}
	sol = max(curr,sol);

	cout<<sol<<endl;

}


int main()
{
	ios_base::sync_with_stdio(0), cin.tie(0), cout.tie(0);
	int t = 1;
	// cin >> t;

	while (t--)
	{
		solve();
	}
	return 0;
}